src/handler.cr

```diff
@@ -5,33 +5,43 @@ require "./users_collection"
 
 # JWT authorization middleware
 class KemalJWTAuth::Handler < Kemal::Handler
-  # Used to encrypt all JWTs
-  private property secret_key : String
-  @logger : Logger
   property sign_in_endpoint = "/sign_in"
   property users : UsersCollection
+  property logger : Logger = Logger.new STDOUT
+  property algorithm : JWT::Algorithm = JWT::Algorithm::HS256
+  # Used to encrypt all JWTs
+  @secret_key : String
 
-  def initialize(@users : UsersCollection, @secret_key = Random::Secure.base64(32), @algorithm = "HS256", @logger = Logger.new STDOUT); end
+  # def initialize(@users, @secret_key : String, @algorithm : JWT::Algorithm, @logger : Logger)
+  # end
+
+  # def self.new(users_file : String, secret_key = Random::Secure.base64(32), algorithm = JWT::Algorithm::HS256::HS256, logger = Logger.new STDOUT)
```

Comment:
why are these commented out? Should they be removed?

Reply:
yes sorry